src/pseudo/load_psp.jl

```diff
@@ -28,8 +27,7 @@ function load_psp(key::AbstractString)
         if startswith(identifier, datadir_psp())
             identifier = identifier[length(datadir_psp())+1:end]
         end
-    else
-        # Not a file: Treat as identifier, add extension if needed
+    else  # Not a file or url: treat as identifier, add extension if needed
```

Comment:
Could also be worth integrating better with the Artefact system here, but I suppose we should get to this once PseudoPotentialIO is in.